hackathon/models.py

```diff
@@ -112,15 +114,20 @@ class HackProject(models.Model):
     "scores" has been moved to HackProjectScore. See comments there."""
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
-    # Each model can only be created by one user: One To Many
-    created_by = models.ForeignKey(User,
-                                   on_delete=models.CASCADE,
-                                   related_name="hackprojects")
+    created_by = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="hackproject",
+        default="")
```

Comment:
I think you can remove the `default=""` here, this is only needed for Char or TextField.